src/pages/trade-types/multiplier/_what-are-multipliers.js

```diff
@@ -207,13 +231,14 @@ const WhatAreOptions = () => {
                         <Row mb="32px">
                             <RowColumn>
                                 <ImgWrapper>
-                                    <QueryImage data={data['multiplier_no_multi_loss']} alt="tes" />
+                                    <QueryImage data={is_eu_country ? data['multiplier_no_multi_loss_eu'] : data['multiplier_no_multi_loss']}
+                                    alt="tes" />
                                 </ImgWrapper>
 
                                 <TextWrapper>
                                     <Text m="1.6rem 0 4rem 0">
                                         <Localize
-                                            translate_text="<0>With an equivalent $100 position on CFDs</0>, with 1:500 leverage, you risk 2% * $50,000 = <1>$1,000 loss</1>."
+                                            translate_text={is_eu_country ? "<0>With an equivalent $100 position on CFDs</0>, with 1:5 leverage, you risk 25% * $500 = <1>$125 loss</1>." : "<0>With an equivalent $100 position on CFDs</0>, with 1:500 leverage, you risk 2% * $50,000 = <1>$1,000 loss</1>."}
```

Comment:
I don't think you're gonna be able to extract both the strings here for the translation. You can keep the `is_eu_country` check outside the Localize component. In this case, we're gonna need 2 components for either of the cases.

Reply:
fixed, thanks